apps/src/templates/projects/publishDialog/PublishDialog.story.jsx

```diff
@@ -1,39 +1,31 @@
 import React from 'react';
-import {UnconnectedPublishDialog as PublishDialog} from './PublishDialog';
 import {action} from '@storybook/addon-actions';
+import {UnconnectedPublishDialog as PublishDialog} from './PublishDialog';
 
 const PROJECT_ID = 'MY_PROJECT_ID';
 const PROJECT_TYPE = 'MY_PROJECT_TYPE';
 
-export default storybook => {
-  return storybook.storiesOf('Dialogs/PublishDialog', module).addStoryTable([
-    {
-      name: 'dialog open',
-      description: '',
-      story: () => (
-        <PublishDialog
-          isOpen={true}
-          isPublishPending={false}
-          projectId={PROJECT_ID}
-          projectType={PROJECT_TYPE}
-          onConfirmPublish={action('publish')}
-          onClose={action('close')}
-        />
-      )
-    },
-    {
-      name: 'dialog open with publish pending',
-      description: '',
-      story: () => (
-        <PublishDialog
-          isOpen={true}
-          isPublishPending={true}
-          projectId={PROJECT_ID}
-          projectType={PROJECT_TYPE}
-          onConfirmPublish={action('publish')}
-          onClose={action('close')}
-        />
-      )
-    }
-  ]);
+const publishDialogDefaultProps = {
+  isOpen: true,
+  isPublishPending: false,
+  projectId: PROJECT_ID,
+  projectType: PROJECT_TYPE,
+  onConfirmPublish: action('publish'),
+  onClose: action('close')
+};
+
+const Template = args => <PublishDialog {...args} />;
+
+export const DialogOpen = Template.bind({});
+DialogOpen.args = {...publishDialogDefaultProps};
```

Comment:
for default props like this, you can include them in the template so that each story automatically has them and can override any they want to change. it would look something like this:

```javascript
const Template = args => <PublishDialog {...publishDialogDefaultProps} {...args} />;
```